reference/array/functions/extract.xml

```diff
@@ -1,220 +1,247 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!-- EN-Revision: n/a Maintainer: leszek Status: ready -->
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: 4a7ddddc27271967b616ad3400cfbe2a9b48212b Maintainer: leszek Status: ready -->
 <!-- $Revision$ -->
+<!-- CREDITS: sobak -->
```

Comment:
One thing of note, is that we don't really use the CREDITS comments anymore since git allows us to give proper authorship. :)

Reply:
Ah, true that. I'm not really sure why did I even add it here - it's probably a reflection of me being torn apart between just reassigning myself to be a maintainer and crediting the original author enough :)